docs/user/dashboard/url-drilldown.asciidoc

```diff
@@ -137,7 +137,7 @@ Example:
 
 
 [float]
-[[variables]]
+[[url-template-variables]]
```

Comment:
This change means we also have to change: https://github.com/elastic/kibana/blob/master/src/core/public/doc_links/doc_links_service.ts#L44

I think it is fine to fix `docsLinks` in separate follow up pr to make current pr move faster through backports (to no trigger whole CI because of code changes) as it's anyway won't hurt 7.10 where we want to fix different problem.
Leaving this up to you if you'd like to make this change in current pr